install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+# This script downloads the latest release of run from GitHub and installs it 
+# to the current directory.
+
+# Print an error message before exiting.
+trap 'echo "Error: $0:$LINENO -- installation failed"' ERR 
+
+set -e          # Exit immediately if a command fails
+set -o pipefail # Fail a pipeline if any command fails
+
+main() {
+    # Check for the existence of a file named "run" in the current directory.
```

Comment:
Please check for a directory named run also. 

my mnemonic:
- `-f` checks for <em><strong>f</strong></em>iles,
- `-d` checks for <em><strong>d</strong></em>irs,
- `-e` checks for <em><strong>e</strong></em>ither.